netlify.toml

```diff
@@ -24,4 +24,257 @@
 [[headers]]
   for = "/*.json"
     [headers.values]
-    Access-Control-Allow-Origin = "*"
\ No newline at end of file
+    Access-Control-Allow-Origin = "*"
+
+# Legacy Website Redirects
+
+# getting started
+[[redirects]]
+  from = "/documentation/get-started"
+  to = "/get-started/"
+  status = 200
+
+# news -> releases
+[[redirects]]
+  from = "/news/*"
+  to = "/releases/v4"
+  status = 200
+
+# community ->
+[[redirects]]
+  from = "/news/*"
+  to = "/releases/v4"
+  status = 200
+
+# Icons
+[[redirects]]
+  from = "/icons"
+  to = "/foundation/icons"
+  status = 200
+
+# foundation pages
+[[redirects]]
+  from = "/documentation/accessibility"
+  to = "/foundation/accessibility/"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/app-layout"
+  to = "/foundation/app-layout/"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/color"
+  to = "/foundation/color/"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/internationalization"
+  to = "/foundation/internationalization/"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/navigation"
+  to = "/foundation/navigation/"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/themes"
+  to = "/foundation/themes/"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/typography"
+  to = "/foundation/typography/"
+  status = 200
+
+# Redirect components
+[[redirects]]
+  from = "/documentation/accordion"
+  to = "/components/accordion"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/alerts"
+  to = "/components/alert"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/badges"
+  to = "/components/badge"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/buttons"
+  to = "/components/button"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/button-group/"
+  to = "/components/button-group"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/cards"
+  to = "/components/card"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/checkboxes"
+  to = "/components/checkbox"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/datagrid/*"
+  to = "/components/datagrid"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/datalist"
+  to = "/components/datalist"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/datepicker"
+  to = "/components/date-picker"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/dropdowns"
+  to = "/components/dropdown"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/forms"
+  to = "/components/form"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/grid"
+  to = "/components/grid"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/header"
+  to = "/components/header"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/input"
+  to = "/components/input"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/labels"
+  to = "/components/label"
+  status = 200
+
+[[redirects]]
+  from = "/documentation/list"
```

Comment:
Using the `_redirects` file might be easier to keep the config cleaner. Also these should all be 301.

Then we should only have to list the specific paths that have a name change (pluralization issue).

```
/documentation/labels /components/label 301 # Specific because path name changed
/documentation /components 301 # Catch all for any that didn't change
```

Reply:
This was fixed in https://github.com/vmware/clarity/pull/4820/commits/822736a8e9076a6d934a53022f46ee5da1798d3a